server/src/clients/google.py

```diff
@@ -92,3 +92,38 @@ def get_user_info(
             parsed_user_info[field] = user_info.get(field, "")
 
         return (None, parsed_user_info)
+
+
+@error_bounded(
+    (AppError(ErrorCode.GOOGLE_API_ERROR, "Google api error - get user contacts"), None)
+)
+def get_people_connections_list(token: str,) -> Tuple[Optional[AppError], List[object]]:
+    """
+    Request list of user's contacts from Google People API.
+
+    Returns a list of Google API Person objects representing all
```

Comment:
What's the shape of these Google API person objects? Could you create a data class to represent them - so it's easier for the rest of the code to use them. https://docs.python.org/3/library/dataclasses.html

Reply:
The individual "people" look like this, and it's kind of gross (they all just map to lists because they could have more than one entry for almost every property lol):

https://developers.google.com/people/api/rest/v1/people#Person

Reply:
And also all of the lists just contain multiple of even more Google API object classes lol, for ex. email:

https://developers.google.com/people/api/rest/v1/people#Person.EmailAddress